tests/mappers/test_exon_genomic_coords.py

```diff
@@ -1307,6 +1298,7 @@ async def test_wee1(test_egc_mapper, wee1_exon2_exon11, mane_wee1_exon2_exon11):
         "seg_start_genomic": 9597639,  # GRCh38 coords: 9576092
         "seg_end_genomic": 9609996,  # GRCh38 coords: 9588449
         "gene": "WEE1",
+        "starting_assembly": Assembly.GRCH37.value,
```

Comment:
So some of these tests have GRCh38 coords. Maybe update? Didn't look too closely, but did we do a check for GRCh38(Is this necessary?)?

Reply:
The `genomic_to_tx_segment` method assumes the starting assembly is GRCh38 by default, so I think it's fine to not specify this parameter unless we have GRCh37 coordinates? We did not do a check for GRCh38 - the method assumes that the coordinates are derived from the assembly that the user provides.

Reply:
To clarify, "So some of these tests have GRCh38 coords." I mean in this example we have:

```python
        "seg_start_genomic": 9597639,  # GRCh38 coords: 9576092
        "seg_end_genomic": 9609996,  # GRCh38 coords: 9588449
```

Pointing out the comments where we put GRCh38 coords. Since we have this commented, shouldn't starting assembly be GRCh38? Then we will ensure both GRCh37 and GRCh38 are tested.

By "but did we do a check for GRCh38(Is this necessary?)?", I meant, are we testing GRCh38 starting assembly somewhere? This is where I didn't do a check. But if we change the above, then we will have tested GRCh38

Reply:
I think the starting assembly for this should still be GRCh37 since that example is using GRCh37 coordinates. However, I think it would be good to add another example to the `wee1` test which replaces the GRCh37 coordinates with GRCh38 coordinates to ensure that we are testing GRCh38 too.

Reply:
Oh, I realize I made that comment for mappings to GRCh38 coords